rust_snuba/src/consumer.rs

```diff
@@ -130,6 +131,13 @@ pub fn consumer_impl(
         Some(broker_config),
     );
 
+    let runtime = tokio::runtime::Builder::new_multi_thread()
+        .worker_threads(concurrency)
+        .enable_all()
+        .build()
+        .unwrap();
+    let concurrency = ConcurrencyConfig::with_runtime(concurrency, runtime.handle().to_owned());
```

Comment:
Is there a difference between this and calling `ConcurrencyConfig::new(concurrency)`?

Reply:
you are absolutely right, I just switched to `new`